### DISCUSSION TITLE
Practicas

### DISCUSSION THREAD

**Discussion by DavidNghtmr** [Launch-X-Latam/MisionFrontEnd#418] [Show and tell]
Estas son mis practicas para la mision FrontEnd 
https://github.com/DavidNghtmr/launchx-frontend

**Comment by OscarJG20** [comment]
Hola!
Creo que tus actividades van bastante bien, en el caso del apartado de "Preparations Before Vaccine" en la página de vacunas le podrías agregar un blur o un transparent al contenedor de cada uno para que se aprecie mejor las letras.

El estilo de tu pokedex me gustó bastante, tal vez no hiciste eso a mano, pero sin duda supiste combinar las cosas y hace que se vea muy bien, para dispositivos móviles puedes considerar centrar todo el contenido de manera vertical, pero muy bien hecho, sigue así!

Chaito :D
